build-docc.sh

```diff
@@ -0,0 +1,15 @@
+##!/bin/sh
+
+xcrun xcodebuild docbuild \
```

Comment:
might be nice to move this into a subfolder like `tools` so tat it is "hidden" for those who don't need to know about it

Reply:
How about folder named `scripts`?

Reply:
I'm going to leave it for now, it's fine